content/soda-gettingstarted/multicloud-local-installation-using-ansible.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Multi-cloud Local Cluster Installation using Ansible
+menuTitle: ""
+disableToc: true
+tags: ["user guide", "multi-cloud", "ansible"] 
```

Comment:
Remove tag 'user guide' and add tag "installation guide"

Reply:
Fixed both issues